.github/workflows/pr-checker.yml

```diff
@@ -63,6 +63,18 @@ jobs:
       run: fw-tools/edge-testnet
     - name: Run edge-testnet as fake snap
       run: SNAP=snap fw-tools/edge-testnet
+    - name: Check -h works OK.
+      run: |
+        if ! fw-tools/edge-testnet -h; then
+          echo "Failed to run edge-testnet -h"
+          exit 1
+        fi
+    - name: Check if it works with a custom domain
+      run: |
+        if ! fw-tools/edge-testnet --domain pdm-sandbox.io; then
```

Comment:
for the pr-tester, the domain name should be `-pr-tester.pdm-sandbox.io`

Reply:
Fixing it.